### DISCUSSION TITLE
A theme that shows number of staged and unstaged git changes?

### DISCUSSION THREAD

**Discussion by birgersp** [ohmyzsh/ohmyzsh#10948] [Help] [answered]
I want to see number of staged/unstaged changes. Like in posh-git:

![image](https://raw.githubusercontent.com/wiki/dahlbyk/posh-git/images/PromptDefaultLong.png)

**Comment by carlosala** [comment, marked as answer]
This can be done with the plugin `git-prompt`, and editing a per-default theme!
